etc/column/column.conf

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+# Log file
+log_file = /var/log/column.log
+
+# Log info level
+log_level = DEBUG
+
+[API]
+server = 0.0.0.0
```

Comment:
Don't listen on address 0.0.0.0 (all interfaces) by default. This can be a potential security issue if someone doesn't realize the service is bound to an insecure interface.